requirements.txt

```diff
@@ -8,9 +8,3 @@ pymbar==3.0.5
 pyyaml==5.4.1
 hilbertcurve==1.0.5
 networkx==2.5
-
-# Development/Testing Reqs
-black
-pytest
-pytest-cov
-pre-commit
```

Comment:
Dev and testing dependencies can be installed via `pip install .[dev]` or `pip install .[test]` respectively.